dashboard/apps/prototype/migrations/0001_initial.py

```diff
@@ -1,82 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
migrations dir shouldn't get added again unless migrations are initialised. can do that when we have a more stable model.